hibernate-core/src/main/java/org/hibernate/dialect/function/StandardSQLFunction.java

```diff
@@ -28,8 +28,8 @@ public StandardSQLFunction(String name, AllowableFunctionReturnType type) {
 		this( name, true, type );
 	}
 
-	public StandardSQLFunction(String name, boolean useParentheses, AllowableFunctionReturnType type) {
-		super( name, useParentheses, null, null );
+	public StandardSQLFunction(String name, boolean requiresArguments, AllowableFunctionReturnType type) {
+		super( name, name, requiresArguments, null, null, null);
```

Comment:
super minor, but it seems a trailing empty char at the end would be consistent with existing coding style